doc/data_format.rst

```diff
@@ -596,6 +596,47 @@ Version history
 - 5 (20141010) Added more items.
 
 
+Fixed gentle and thrill rides
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Gentle and thrill rides consisting of a single building. FreeRCT can read block version 1.
+
+======  ======  =======  ===================================================================================
+Offset  Length  Version  Description
+======  ======  =======  ===================================================================================
+   0       4      1-     Magic string 'FGTR'.
+   4       4      1-     Version number of the block.
+   8       4      1-     Length of the block excluding magic string, version, and length.
+  12       1      1-     Magic number 1 for thrill rides or 0 for gentle rides.
+  13       2      1-     Zoom-width of a tile of the surface.
+  15       1      1-     Length of the ride in x direction in voxels.
+  16       1      1-     Length of the ride in y direction in voxels.
+  17       x      1-     Heights of the ride in voxels. The number x of height bytes is
+                         equal to the product of the ride lengths in x and y direction.
```

Comment:
Having `x` and `y` directions, as well as an `x` in the length column of this field is confusing. Instead of `x` use a different letter to avoid it, for example `s` (size) or `c` (count) but any other letter will do except `x` and `y`.

The explanation is a bit short, perhaps `Heights of the ride in voxels, for each tile occupied by the ride.` ? (ie also explain why you need `x` bytes).

Explanation about value `x` can be added here or below the table.